<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PermSettings</class>
 <widget class="QDialog" name="PermSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>KCloud::Permessi</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>100</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <row>
      <property name="text">
       <string>New Row</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Utente</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Permesso</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>inserisci email</string>
      </property>
     </item>
     <item row="0" column="1">
      <property name="flags">
       <set>ItemIsSelectable|ItemIsEnabled</set>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="publicChec">
       <property name="text">
        <string>Rendi Pubblico</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="readRadio">
       <property name="text">
        <string>Lettura</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="writeRadio">
       <property name="text">
        <string>Scrittura</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="text">
        <string>Annulla</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delUserBtn">
       <property name="text">
        <string>Rimuovi</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addUserBtn">
       <property name="text">
        <string>Aggiungi</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="confirmBtn">
       <property name="text">
        <string>Conferma</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
